### PR TITLE
[Documentation] Add extra help information on serializer id errors

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

**Comment by Arkatufus** [comment]
Written documentation is not complete yet, code is complete.

File: docs/articles/toc.yml [hunk 1]
```diff
@@ -202,3 +202,5 @@
     href: utilities/circuit-breaker.md
 - name: Configurations
   href: configuration/toc.yml
+- name: Serialization
+  href: serialization/toc.yml
```
Comment (Aaronontheweb) [inline docs/articles/toc.yml:206]: We should probably migrate some stuff to this section

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by Aaronontheweb** [comment]
This should help quite a bit when debugging serialization configuration errors.

File: docs/articles/toc.yml (hunk 1 above)
Comment (Arkatufus) [reply docs/articles/toc.yml:206]: There's a plan on moving Hyperion to its own page, so I think this section would host it.

**Pull request merged**